src/calculateRentalCost.js

```diff
@@ -1,10 +1,13 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
```

Comment:
Consider using a constant for the daily rental cost instead of hardcoding the value 40. This will make your code more maintainable and understandable.